card/mod/01_core/set/all/name_validations.rb

```diff
@@ -0,0 +1,66 @@
+event :permit_codename, :validate,
+      on: :update, changed: :codename do
+  errors.add :codename, 'only admins can set codename' unless Auth.always_ok?
+  validate_unique_codename
+end
+
+event :validate_unique_codename do
+  return unless codename.present? && errors.empty? &&
+     Card.find_by_codename(codename).present?
```

Comment:
Align the operands of a condition in an `unless` statement spanning multiple lines.